ui/static_table/viewmodels/OBATableRow.m

```diff
@@ -23,7 +23,7 @@ + (void)load {
 }
 
 + (instancetype)tableRowWithTitle:(NSString*)title action:(void (^)())action {
-    return [[self alloc] initWithTitle:title action:action];
+    return [[OBATableRow alloc] initWithTitle:title action:action];
```

Comment:
please revert. Using `self` instead of `OBATableRow` means that the proper class is `alloc`d regardless of where this is called from. Imagine, e.g., that you have a subclass of `OBATableRow` called `ChadTableRow`. If you call `+[ChadTableRow tableRowWithTitle:action:]`, with this change, you'll end up getting back an instance of type `OBATableRow` unexpectedly.